scripts/hooks/hook-googleapiclient.model.py

```diff
@@ -0,0 +1,5 @@
+"""Temporary fix for https://github.com/iterative/dvc/issues/5618."""
+from PyInstaller.utils.hooks import collect_data_files, copy_metadata
```

Comment:
```suggestion
from PyInstaller.utils.hooks import (    # pylint:disable=import-error
    collect_data_files, copy_metadata
)
```

pylint isn't aware of the pyinstaller imports, you can just suppress the linter error here (see the other hook scripts)

Reply:
🙏 Thank you very much, @pmrowla!

Reply:
There's a new linter issue - imports should be on separate lines (my mistake). You should set up our pre-commit hooks so this shows up in your environment though (along with things like the original pylint issue) https://dvc.org/doc/user-guide/contributing/core#development-environment